### DISCUSSION TITLE
Select Net Routing

### DISCUSSION THREAD

**Discussion by davidcarails** [The-OpenROAD-Project/OpenROAD#2141] [Q&A] [answered]
Is there a mechanism I can use to select which nets to route or not route with the global and detail routers?

**Comment by maliberty** [comment, marked as answer]
There is no user control currently.  The only trick you could play is to temporarily mark the nets not to route as special nets and then rest when done routing.  

What is your use case?

**Reply by davidcarails** [reply]
I am trying to integrate the open road router into another cad environment.
